packages/@sanity/field/src/diff/components/DiffTooltip.tsx

```diff
@@ -77,16 +77,26 @@ function AnnotationItem({annotation}: {annotation: AnnotationDetails}) {
   }
 
   return (
-    <div className={styles.annotation}>
-      <div className={styles.user} style={{backgroundColor: color.background, color: color.text}}>
-        <span>
-          <UserAvatar userId={author} />
-        </span>
-        <span className={styles.displayName}>{user ? user.displayName : 'Loading…'}</span>
-      </div>
-      <time className={styles.timeAgo} dateTime={timestamp}>
+    <Inline space={2}>
+      <Flex
+        align="center"
+        paddingRight={3}
+        style={{
+          backgroundColor: color.background,
+          color: color.text,
+          borderRadius: 'calc(23px / 2)',
+        }}
```

Comment:
Nitpick: I believe we could make this more reusable by creating a styled component here. Something like:

```tsx
const UserColoredCard = styled(Card)<{$color: {background: string, text: string}}>`
  --card-bg-color: ${props => props.$color.background};
  --card-fg-color: ${props => props.$color.text};
  --card-muted-fg-color: ${props => props.$color.text};
`
```

And using it like:
```tsx
<UserColoredCard $color={color} style={{borderRadius: 'calc(23px / 2)'}}>
  ...
</UserColoredCard>
```